sys-apps/plocate/plocate-1.1.5.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info meson systemd
+
+DESCRIPTION="Posting locate is a much faster locate"
+HOMEPAGE="https://plocate.sesse.net/"
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URL="https://git.sesse.net/plocate"
+else
+	SRC_URI="https://plocate.sesse.net/download/${P}.tar.gz"
+fi
+
+LICENSE="GPL-2 GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="io-uring"
+
+RDEPEND="
+	acct-group/locate
+	app-arch/zstd:=
+	!sys-apps/mlocate
+	io-uring? ( sys-libs/liburing:= )
+"
+DEPEND="${RDEPEND}"
+
+BDEPEND="acct-group/locate"
```

Comment:
DEPEND+RDEPEND, no? (DEPEND given it's needed by the build system, it seems). Don't think we need it for BDEPEND?